src/main/java/com/indeed/jiraactions/ActionFactory.java

```diff
@@ -30,33 +40,33 @@ public ActionFactory(final UserLookupService userLookupService,
     }
 
     public Action create(final Issue issue) throws IOException {
```

Comment:
Not sure it makes sense to have these methods that create full `Action` objects when you only ever one of them. Feels like you just want some kind of mutable state. (Or just collect the Actions as usual and then aggregate them to create a single Issue object.)